src/Tools/XmlDocumentationsUtility/Test/XmlDocUtilityTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using XmlDocumentationsUtility;
+
+namespace XmlDocumentationsUtility.Test
+{
+    [TestFixture]
+    class XmlDocUtilityTests
```

Comment:
Can you also add one more test for RemoveDocumentationForHiddenNodes() method and ensure that after removing hidden nodes, the new xml has all the nodes which was required.